eth/options.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Celer Network
+
+package eth
+
+import (
+	"math/big"
+	"time"
+)
+
+type txOptions struct {
+	ethValue           *big.Int      // transact
+	minGasGwei         uint64        // transact
+	maxGasGwei         uint64        // transact
```

Comment:
nit: consider add a field plusGasGwei meaning add this much on top of recommended gas.
and I don't think maxGasGwei ever helps, instead we have to constantly update rtconfig due to congested mainnet.
or if we say this is to safe guard any mistake setting ridiculous gas price, then maxGasGwei should be really large like 100 or even higher

Reply:
Added the new field. Agree about maxGasGwei, we should keep this and set to a very large value in rtconfig.